phalcon/paginator/adapterinterface.zep

```diff
@@ -33,9 +33,16 @@ interface AdapterInterface
 
 	/**
 	 * Returns a slice of the resultset to show in the pagination
+	 *
+	 * @deprecated will be removed after 4.0
 	 */
 	public function getPaginate() -> <\stdClass>;
 
+	/**
+	 * Returns a slice of the resultset to show in the pagination
+	 */
+	public function paginate() -> <\stdClass>;
```

Comment:
For not breaking BC(like someone implementing their own adapter) we shouldn't add method to interfaces right now.

Reply:
Gotcha. I will make the necessary changes to the interface for 3.4.x